proto/auction/v1/query.proto

```diff
@@ -85,8 +88,14 @@ message QueryBidResponse {
 
 message QueryBidsByAuctionRequest {
   uint32 auction_id = 1;
+  Pagination pagination = 2;
 }
 
 message QueryBidsByAuctionResponse {
   repeated Bid bids = 1;
+  uint64 next_page = 2;
+}
+
+message Pagination {
```

Comment:
I can't find the comment from Collin's review, but yes, there are expected ways for pagination to function for Cosmos queries. I would recommend implementing the SDK version of this.